Apps/W1/HybridBaseDeployment/app/src/codeunits/HybridCloudManagement.Codeunit.al

```diff
@@ -941,6 +942,14 @@ codeunit 4001 "Hybrid Cloud Management"
         OnInsertDefaultTableMappings(IntelligentCloudSetup."Product ID", DeleteExisting);
     end;
 
+    procedure PrepareMigrationValidation()
+        IntelligentCloudSetup: Record "Intelligent Cloud Setup";
+    begin
+        if IntelligentCloudSetup.Get() then;
```

Comment:
we need a hard error here. We should not raise an event and not know in which data the setup is